tests/Test/AbstractFixerTestCase.php

```diff
@@ -166,6 +166,14 @@ final public function testFixerDefinitions(): void
                 }
                 self::assertTrue($hasSuitableSupportedVersion, 'Version specific code sample must be suitable for at least 1 supported PHP version.');
 
+                $hasUnsuitableSupportedVersion = false;
+                foreach ($supportedPhpVersions as $version) {
+                    if (!$sample->isSuitableFor($version)) {
+                        $hasUnsuitableSupportedVersion = true;
+                    }
+                }
+                self::assertTrue($hasUnsuitableSupportedVersion, 'Version specific code sample must be unsuitable for at least 1 supported PHP version.');
```

Comment:
```suggestion
                self::assertTrue($hasUnsuitableSupportedVersion, 'Version specific code sample must specify version which is higher than lowest supported PHP version');
```

I find current message not intuitive, maybe it would be better like this?

Reply:
The new one is not true, there are code samples that are limited by highest PHP version allowed, like [this](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.23.0/src/Fixer/NamespaceNotation/CleanNamespaceFixer.php#L33) one.

Reply:
OK, I did not remember if there's also upper limit. Then this message must suffice 🤷‍♂️.